Tasks/Gradle/CodeAnalysis/checkstyle.gradle

```diff
@@ -16,8 +16,12 @@ rootProject {
 
                 proj.apply plugin: 'checkstyle'
                 checkstyle {
-                    println(rootDir);
-                    configFile rootProject.file('checkstyle.xml')
```

Comment:
This seems to be resolved differently on windows vs linux. So much cross platform :)